CodeStarterCamp_Week4/main.swift

```diff
@@ -80,52 +86,174 @@ class BodyCondition {
     }
 }
 
+func checkInputCount (inputCountRoutines: String?) throws -> Int {
+    var returnInt = 0
+    if let inputToString = inputCountRoutines, let inputToInt = Int(inputToString) {
+            guard inputToInt >= 1 && inputToInt < 100 else {
+                throw ActivityError.invalidInput
+            }
+            returnInt = inputToInt
+        } else {
+            throw ActivityError.invalidInput
+        }
+    
+    return returnInt
+}
+
+func doActivity (bodyCondition: BodyCondition, activityName: String, 상체근력: ClosedRange<Int>, 하체근력: ClosedRange<Int>, 근지구력: ClosedRange<Int>, 피로도: ClosedRange<Int>) {
+    print("<<\(activityName)을(를) 시작합니다>>")
+    bodyCondition.상체근력 += Int.random(in: 상체근력)
+    bodyCondition.하체근력 += Int.random(in: 하체근력)
+    bodyCondition.근지구력 += Int.random(in: 근지구력)
+    bodyCondition.피로도 += Int.random(in: 피로도)
+    print("--------------")
+}
```

Comment:
```suggestion
func doActivity (
    bodyCondition: BodyCondition, 
    activityName: String, 
    상체근력: ClosedRange<Int>, 
    하체근력: ClosedRange<Int>, 
    근지구력: ClosedRange<Int>, 
    피로도: ClosedRange<Int>
) {
    print("<<\(activityName)을(를) 시작합니다>>")
    bodyCondition.상체근력 += Int.random(in: 상체근력)
    bodyCondition.하체근력 += Int.random(in: 하체근력)
    bodyCondition.근지구력 += Int.random(in: 근지구력)
    bodyCondition.피로도 += Int.random(in: 피로도)
    print("--------------")
}
```

파라미터가 많아지면 오른쪽으로 늘리는 대신, 가독성을 위해 줄바꿈을 이렇게 해주기도 합니다. ☺️
또한 함수명 뒤 소괄호의 시작 사이에 공백이 있는데 보통 공백은 넣어주지 않는 것 같아요 🧐